WalletWasabi/Helpers/HttpMessageHelper.cs

```diff
@@ -454,7 +441,7 @@ public static void ParseFistChunkLine(string firstChunkLine, out long chunkSize,
 			}
 
 			var allData = new byte[(int)length];
-			var num = await stream.ReadBlockAsync(allData, (int)length);
+			var num = await stream.ReadBlockAsync(allData, (int)length, ctsToken);
```

Comment:
Not sure about this.

Reply:
Ok, then we could remove the ctsToken from the parameters list.

Reply:
Yes, please remove it.